core_backend/tests/api/test.env

```diff
@@ -8,3 +8,4 @@ QDRANT_PORT=6334
 
 LLM_MODEL="gpt-3.5-turbo"
 PROMETHEUS_MULTIPROC_DIR=/tmp
+ALIGN_SCORE_API="http://localhost:5002/alignscore_base"
```

Comment:
Only used when running locally

Reply:
/nit the 5002 here + in the Makefile vs 5001 everywhere else is a bit confusing. I assume you changed this so there wouldn't be conflicts if the person has the app running locally while trying to run pytest at the same time - it makes sense to me but is there a cleaner way? I've seen some apps say "port 5001 is busy, running on port 5002" so it's automated.

Reply:
Yes that's right. Note that we do the same for QDRANT_PORT and POSTGRES_PORT in this file. 
The point of having these env var is to allow us to change it. Switching ports automatically within docker containers is not easy! We can add it as a nice to have though.